src/components/ui/Calendar.tsx

```diff
@@ -0,0 +1,68 @@
+'use client';
+
+import { ChevronLeft, ChevronRight } from 'lucide-react';
+import type * as React from 'react';
```

Comment:
importing React is unnecessary. Nextjs does it automatically

Reply:
nvm lol this is a component from shadcn maybe we ignore it?

Reply:
did we use Icon prefix in the others?

Reply:
Will make a separate PR to remove unnecessary imports for shadcn components